test/functional/home_controller_test.rb

```diff
@@ -59,10 +59,10 @@ def setup
     @wikis.each do |obj|
       if obj.class == Revision && obj.status == 1
         selections = css_select '.wiki'
-        assert_equal selections.length, 7
+        assert_equal 9, selections.length
```

Comment:
The order of arguments wasn't right. It should be expected value first. (Also updated the number because I added more to the fixtures.)